package.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+# Usage: ./package.sh $tar_ball_name $bin_path
+# This script will create a tar ball for the specified arch.
+
+set -e
+set -x
+
+DESTINATION="packaging"
+WORK_DIR="_packaging_internal"
+
+tar_ball_name=$1
+bin_path=$2
+
+if [ -z "$tar_ball_name" ]
+then
+  echo "ERROR: ./package.sh requires the first positional param to be the name of the tar ball."
+  exit 1
+fi
+
+if [ -z "$bin_path" ]
```

Comment:
Nit: Do you also want to check that $bin_path exists on the filesystem?

Reply:
Sorry what do you mean? I thought this line is checking if the $bin_path exist? Btw I'm merging this now but I will have another commit coming cause I want to at least make sure I am able to build this since I messed up with the `sync fork`.

Reply:
`-z` check if the variable is undefined or empty iirc. You need `-e` (or `-f` if you want to specifically check for file and not allow directories etc). Ok to merge this PR and address it later.
If `-z` was checking for existence, it would have failed for the first check because the tarball-to-be-created doesn't exist yet